src/components/Router/index.js

```diff
@@ -303,8 +304,8 @@ class HistoryRouter extends Component {
 
   buildComponentsList =
     ({flow},
-    {documentType, steps, options: {mobileFlow}}) =>
-      componentsList({flow, documentType, steps, mobileFlow});
+    {documentType, hasPreselectedDoc, steps, options: {mobileFlow}}) =>
```

Comment:
the information of whether there is a preselected document is in the `options` isn't it? Not sure why it needs to be passed as an additional arguments. Feels redundant.